packages/shared/src/components/post/BasePostContent.tsx

```diff
@@ -29,20 +30,18 @@ export function BasePostContent({
   customNavigation,
 }: BasePostContentProps): ReactElement {
   const { id } = post ?? {};
+  const { onCloseShare, sharePost, onSharePost } = engagementProps;
+  const { onStartArticleOnboarding, showArticleOnboarding } =
```

Comment:
Glad another of thesee is removed

`    // @NOTE see https://dailydotdev.atlassian.net/l/cp/dK9h1zoM
    // eslint-disable-next-line react-hooks/rules-of-hooks`